packages/storage/src/types.ts

```diff
@@ -10,6 +10,9 @@ export type ReferenceObject = {
   tags?: string[];
   extra?: Trait[];
   history?: string[];
+  creatorAddress?: string;
```

Comment:
This looks like it comes from a renaming in GraphQL, and no something stored to arweave. The `storage` module is meant for storing stuff on arweave. So you might want to introduce a `ReferenceObject & {creatorAddress; metadata_content_flag; id}` type on the consumer side, but here is not the right place for it.